src/clients/storage/InternalStorageClient.cpp

```diff
@@ -131,5 +131,46 @@ cpp2::ChainAddEdgesRequest InternalStorageClient::makeChainAddReq(const cpp2::Ad
   return ret;
 }
 
+void InternalStorageClient::chainDeleteEdges(cpp2::DeleteEdgesRequest& req,
+                                             const std::string& txnId,
+                                             TermID termId,
+                                             folly::Promise<nebula::cpp2::ErrorCode>&& p,
+                                             folly::EventBase* evb) {
+  auto spaceId = req.get_space_id();
+  auto partId = req.get_parts().begin()->first;
+  auto optLeader = getLeader(req.get_space_id(), partId);
+  if (!optLeader.ok()) {
+    LOG(WARNING) << folly::sformat("failed to get leader, space {}, part {}", spaceId, partId);
+    p.setValue(::nebula::cpp2::ErrorCode::E_SPACE_NOT_FOUND);
```

Comment:
The error code in this place is not appropriate, including `InternalStorageClient::chainAddEdges`

Reply:
actually, the status returned may be "space_not_found", or "part_not_found". 
normally, neither would happen here. 
(if this really happen, we can't get here, should stop before)
But we may occur drop space during toss execution, thus I return space_not_found.